doc/GOALS.md

```diff
@@ -15,7 +15,7 @@ This helps in achieving the following goals:
   - Uniformity of configuration access (logging, vcs commit, notifications).
   - Syntax of the configuration files (with limitations, see below).
 
-Elektra follows following goals, in order of preference.
+Elektra follows goals, in order of preference.
```

Comment:
I don't understand the sentence anymore if we drop "following", maybe you want:

```suggestion
Elektra follows the goals below, in order of preference.
```

Reply:
Thanks for the suggestion!
I had the word repetition ("follows following") in mind for this correction.
Your suggestion is definitely a better and clearer formulation.